index.bs

```diff
@@ -1976,52 +1976,23 @@ need to be careful when exposing these to the web platform.
+Issue: is this advice still current? When was the last time we saw a review with a speculative polyfill?
 
-* Being "polyfillable" is not essential but <strong>is</strong> beneficial
+Polyfills can be hugely beneficial
+in helping to roll out new features to the web platform.
+The Technical Architecture Group <a href="https://www.w3.org/2001/tag/doc/polyfills/">finding on Polyfills and the Evolution of the Web</a>
+offers guidance that should be considered in the development of new features, notably:
+
+* Being "polyfillable" isn't essential but <strong>is</strong> beneficial
 * [[#feature-detect]]
 * Polyfill development should be encouraged
 
 <h3 id="extend-manifests">Extend existing manifest files rather than creating new ones</h3>
 
-New web features should be self-contained and self-describing and ideally should not require an additional manifest file.
-Manifest files can add complexity to the development and deployment process for web technologies.  Sometimes manifest
-files are required, but the use of different manifests on the web should be kept to a minimum.
-
-You may need to make a new manifest file if the domain of the manifest file is different from the existing manifest files.
-For example, if the fetch timing is different, or if the complexity of the manifest warrants it.  Application metadata should be added
-to the Web App Manifest or be an extension of it.
-Manifests designated to be used for specific applications or which require interoperability with non-browsers may need to take a different approach.
-Payment Method Manifest, Publication Manifest, and Origin Policy are examples of these cases.
-
-For example, if you have a single piece of metadata, even if the fetch timing is different than an existing manifest,
-it is probably best to use an existing manifest (or ideally design the feature in such a way that a manifest is not required).
-However, if your feature requires a complex set of metadata specific to a functional domain, the creation of a new manifest may be justified.
-
-Note that in all cases, the naming conventions should be harmonized (see section 2.2)
-
-Note: By principle, existing manifests use lowercase, underscore-delimited names.
-There have been times where it has been beneficial to re-use dictionaries from a manifest in DOM APIs as well, which are camel-cased.
-One such example is the <a href="https://w3c.github.io/image-resource/">image resource</a>. For this reason, if a
-key can clearly be expressed as a single word, that is recommended.
-
-When designing new keys and values for a manifest, make sure they are needed (that is, they enable well-thought-out use-cases).
-Also, please check if a similar key exists. If an existing key/value pair does more or less what is needed,
-work with the existing spec to extend it to your use-case if possible.
+If your feature requires a manifest,
+investigate whether you can extend an existing manifest schema.
 
```

Comment:
Seems like some of the these above points were not captured in the edit. E.g. the thing about fetch timing. Also the list of different kinds of already existent manifest files.

Reply:
Four major existing manifest files are listed below - do we learn any more from the original text?

Regarding fetch timing - I guess I didn't understand the point it was making, and didn't feel that it added anything to the main point. Do we have an example of where a new manifest type was needed because of fetch timing?

Reply:
@hober can we get your thoughts on this please?